src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetUptime.cs

```diff
@@ -0,0 +1,71 @@
+/********************************************************************++
+Copyright (c) Microsoft Corporation.  All rights reserved.
+--********************************************************************/
+
+using System;
+using System.Management.Automation;
+using System.Diagnostics;
+using System.Management.Automation.Internal;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// This class implements Get-Uptime
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "Uptime", DefaultParameterSetName = "Timespan", HelpUri = "")]
```

Comment:
Very good! 

Fixed.

In GetDateCommand.cs I found:
```powershell
    [OutputType(typeof(string), ParameterSetName = new string[] { "UFormat", "net" })]
    [OutputType(typeof(DateTime), ParameterSetName = new string[] { "net" })]
```
Should we do similarly?